source/includes/fundamentals/code-snippets/CompoundOperatorsIndividualExamples.java

```diff
@@ -49,16 +56,26 @@ private void findOneAndUpdateExample() {
         MongoCollection<Document> collection = getCollection();
         //start findOneAndUpdate-example
         // <MongoCollection set up code here>
+
+        // Creates a projection to exclude the "_id" field from the printed documents
```

Comment:
Issue:

I think printed documents is inaccurate. The projection tells the server which fields to return.

Suggestion:

I think it would be better to use the language "retrieved documents" similar to the descriptions of the projection method used in the [code comments sheet](https://docs.google.com/spreadsheets/d/1gCIeV5Bsp1G69MUNQZm8bYK9UNVSQjJt9T7eUM8aJiE/edit#gid=0):


```suggestion
        // Creates a projection to exclude the "_id" field from the retrieved documents
```